src/main/java/User.java

```diff
@@ -0,0 +1,23 @@
+import java.io.Serializable;
+public class User implements Serializable{
+    private String username;
+    private String password;
+    private String email;
+    boolean verified = false;
+    boolean online = false;
+    public User(String username, String password, String email){
+        this.username = username;
+        this.password = password;
+        this.email = email;
+    }
+    public String getEmail(){
+        return this.email;
+    }
+    public String getUsername(){
+        return this.username;
+    }
+    public String getPassword(){
+        return this.password;
```

Comment:
Hi Madhav, I think having getters for your private attributes makes sense. For user password though, since you made the getPassword method public, this allows any class in any java file/package to use that getter method. For instance, Chat could call getPassword with something like Chat.getUser(index i).getPassword(). So it might be better not to have a getter method for password. Instead, you could perhaps change password to protected instead of private, and group all User-related files into a User package. Then only classes within that package can access password. Let me know if this makes any sense!